lib/phases/cold-launch.js

```diff
@@ -116,15 +119,17 @@ ColdLaunch.prototype.capture = function() {
   this.dispatcher.on(Phase.MEMORYENTRY, function(entry) {
     phase.debugEventEntry(Phase.MEMORYENTRY, entry);
 
-    if (entry.context !== app) {
```

Comment:
So it doesn't like reassigning `entry` because we are reassigning only the `entryPoint` property of it?